pkg/migrations/migrations.go

```diff
@@ -35,12 +35,12 @@ type Operation interface {
 // IsolatedOperation is an operation that cannot be executed with other operations
 // in the same migration
 type IsolatedOperation interface {
-	IsIsolated()
+	IsIsolated() bool
```

Comment:
I kind of wanted to avoid having all operations implementing this, it increases the knowledge you need to write an op, while so far this is the only op needing it for real. That's why using duck typing here feels better to me, WDYT?